amundsenatlastypes/schema/01_4_table_report.json

```diff
@@ -0,0 +1,57 @@
+{
+  "entityDefs": [
+    {
+      "superTypes": [
+        "DataSet"
+      ],
+      "name": "TableReport",
```

Comment:
this should be named `Report`

Reply:
changed in 030fe2bce628b835c902856811312dc7a37ae6cb